pgdrive/scene_creator/blocks/block.py

```diff
@@ -449,25 +466,53 @@ def _add_land(self, lanes):
                     theta = -numpy.arctan2(direction_v[1], direction_v[0])
                     width = lane.width_at(0) + self.SIDE_WALK_LINE_DIST * 2
                     length = lane.length
-                    self._add_land2bullet(middle, width, length * 1.3 / segment_num, theta)
-
-    def _add_land2bullet(self, middle, width, length, theta):
-        cm = CardMaker('card')
-        cm.setFrame(-length / 2, length / 2, -width / 2, width / 2)
-        cm.setHasNormals(True)
-        cm.setUvRange((0, 0), (length / 20, width / 10))
-        card = self.card_node_path.attachNewNode(cm.generate())
-        card.setPos(middle[0], -middle[1], numpy.random.rand() * 0.01 - 0.01)
+                    self._add_lane2bullet(middle, width, length * 1.3 / segment_num, theta, lane, (from_, to_, index))
 
-        card.setQuat(
+    def _add_lane2bullet(self, middle, width, length, theta, lane: Union[StraightLane, CircularLane], lane_index):
+        """
+        Add lane visualization and body for it
+        :param middle: Middle point
+        :param width: Lane width
+        :param length: Segment length
+        :param theta: Rotate theta
+        :param lane: Lane info
+        :return: None
+        """
+        segment_np = NodePath(LaneNode(BodyName.Lane, lane, lane_index))
+        segment_node = segment_np.node()
+        segment_node.setActive(False)
+        segment_node.setKinematic(False)
+        segment_node.setStatic(True)
+        shape = BulletBoxShape(Vec3(length / 2, 0.1, width / 2))
+        segment_node.addShape(shape)
+        segment_node.setIntoCollideMask(BitMask32.bit(Block.LANE_SURFACE_COLLISION_MASK))
```

Comment:
Why the `Block.LANE_SURFACE_COLLISION_MASK ` is a static variable of class Block. I think it should goes with Lane or something

Reply:
Actually, it is useless now, since it is in another static physics world. I will remove this.